### ISSUE TITLE
Show notifications without message content

### ISSUE THREAD

**Issue by pigmonkey** [s3rvac/weechat-notify-send#19]
I would like to receive notifications that tell me when a new message has appeared in a buffer, but I do not want the notifications to contain the author or message content itself. For example, I'd like the script to basically execute:

    notify-send "New message in $buffername"

Is it possible for weechat-notify-send to do this?

**Comment by s3rvac** [comment]
This is currently not possible, but I can definitely add support for that into the plugin.

**Comment by s3rvac** [comment]
Implemented in commits 0160a835c60cea64abbbe4242004d8e741154fec and 193dec25af3c9c5dfb5ddd557de4b543bb9725ca. The name of the option is `hide_messages_in_buffers_that_match`. To hide messages in all buffers, execute
```
/set plugins.var.python.notify_send.hide_messages_in_buffers_that_match .*
```
To hide messages only in specific buffers, execute
```
/set plugins.var.python.notify_send.hide_messages_in_buffers_that_match buffer1,buffer2
```

**Comment by pigmonkey** [comment]
Thanks. This is working for my use case.
